src/olympia/devhub/tasks.py

```diff
@@ -139,22 +132,56 @@ def create_version_for_upload(addon, upload, channel):
         add_dynamic_theme_tag(version)
 
 
+@task
+def create_initial_validation_results():
```

Comment:
Need to update `CELERY_TASK_ROUTES` in `settings_base.py` to make sure this task goes to the `devhub` queue, otherwise the chain might not work properly. (Double-check, I think it's also missing `run_wat`)

Reply:
ah yes, thanks